CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## Next Version
+
+- Compiled against [Kotlin 1.3.41](https://github.com/JetBrains/kotlin/releases/tag/v1.3.41)
```

Comment:
I would update to 1.3.61 and coroutines 1.3.2 and it's not full without an update to new Retrofit rewritten on Kotlin

Reply:
I don't think it is necessary anymore to update the dependencies since you are deprecating the library. If you want me to still update this pull requests please let me know.